x/tokenfactory/client/cli/tx.go

```diff
@@ -21,11 +21,19 @@ func GetTxCmd() *cobra.Command {
 		// NewForceTransferCmd(),
 		NewChangeAdminCmd(),
 		NewSetBeforeSendHookCmd(),
+		NewMsgSetDenomMetadata(),
 	)
 
 	return cmd
 }
 
+func NewMsgSetDenomMetadata() *cobra.Command {
+	return osmocli.BuildTxCli[*types.MsgSetDenomMetadata](&osmocli.TxCliDesc{
+		Use:   "set denom metadata",
```

Comment:
thanks for the PR, can we change this to "set-denom-metadata"?

Reply:
ok @PaddyMc